demo/README.md

```diff
@@ -1,13 +1,44 @@
 # ev3dev demo programs
 
-This folder contains a bunch of demo programs that you can use to help you in
+This folder contains several demo programs that you can use to help you in
 developing your own code. Brief descriptions of each demo are provided below;
 you can access the full source code and some more detailed information on each
 by opening the respective folders above.
 
+To install these on your EV3, use git to clone the ev3dev-lang-python repository
+from github. Your EV3 will need Internet connectivty in order to clone the
+repository from github.
+```
+$ sudo apt-get install git
+$ git clone https://github.com/rhempel/ev3dev-lang-python.git
+```
+
+## Running A Program
+There are two ways to run a program.  You can run a program from the command line
+or from the brickman interface.  Note that for both running from the command line
+and running from Brickman the program must be marked as an executable and the
```

Comment:
Can we separate the most important sentence(s) into a new paragraph and bold them?